aws/table_aws_codebuild_build.go

```diff
@@ -144,7 +149,7 @@ func tableAwsCodeBuildBuild(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_JSON,
 			},
 			{
-				Name:        "network_interfaces",
+				Name:        "network_interface",
```

Comment:
Revert the column name to avoid breaking changes, as renaming it could disrupt existing functionality. Instead, applied a `transform.FromField("NetworkInterface")` function based on the API response.